From: coolsnowwolf <coolsnowwolf@gmail.com>
Subject: [PATCH] build: disable manpage generation

The manpages are not installed by the OpenWrt package. Avoid detecting a host
a2x executable whose Python modules are unavailable in the isolated host
environment.

Signed-off-by: coolsnowwolf <coolsnowwolf@gmail.com>
---
--- a/meson.build
+++ b/meson.build
@@ -53,7 +53,7 @@
 add_project_arguments('-DVERSION="'+meson.project_version() + '"', language : 'c')
 
 jose = dependency('jose', version: '>=8')
-a2x = find_program('a2x', required: false)
+a2x = disabler()
 compiler = meson.get_compiler('c')
 
 http_lib = []
